.gitignore

```diff
@@ -1 +1,2 @@
 vendor/
+composer.lock
```

Comment:
I think the lock file should remain in the repo, otherwise we’ll get different versions on each install.

Reply:
Hi @sabberworm, thanks for your reply.

Yes. You're right. And I've reverted this :).